scripts/train/README.md

```diff
@@ -363,7 +374,7 @@ For any HuggingFace model that supports Flash Attention (e.g. Llama and Mistral)
 ```yaml
 model:
     name: hf_causal_lm
-    use_flash_attention_2: True # Will be automatically set to True if Flash Attention V2 is installed and the model supports it
+    attn_implementation: "flash_attention_2"  # Will be automatically set to "flash_attention_2" if Flash Attention V2 is installed and the model supports it
```

Comment:
that automatic comment is just not true. probably my mistake from a while ago, but please delete it